tests/unit/test_amplitude_detuning.py

```diff
@@ -1,90 +1,98 @@
-import tempfile
 from pathlib import Path
 
-import matplotlib
 import pytest
+import tfs
 
 from omc3.amplitude_detuning_analysis import analyse_with_bbq_corrections
-from omc3.plotting.plot_amplitude_detuning import main as pltampdet
 from omc3.plotting.plot_bbq import main as pltbbq
 
-# Forcing non-interactive Agg backend so rendering is done similarly across platforms during tests
-matplotlib.use("Agg")
-
 
 @pytest.mark.basic
```

Comment:
Marked as `basic` but calls a fixture marked as `extended`. Is this wanted?

Reply:
yes. the run with the filter method "outliers" is the basic run (i.e. it tests most of the functionality of the detuning analysis already and if this fails there is something seriously broken). Then the same function is called with different bbq filtering methods, so only that part is different, therefore they are extended. 
If these were proper unit tests, that would look very different ...